docs/application/tizen-studio/release-notes/4-6-release-notes.md

```diff
@@ -0,0 +1,125 @@
+| Web   Widget(Main) | 0                              | 0                             | 0                   | 0                      | 0                         | 0                              |   |
+
+
+  **Hybrid App**
+  Developers can now have multiple dependent applications **of different types** in a single workspace and perform all App life cycle events like app creation, building, packaging, installing and testing.
+    
+| Main App          | Possible Sub App Combinations |
+|-------------------|-------------------------------|
+| Native   Main App | Native + Dotnet               |
+| Dotnet   Main App | Dotnet + Native               |
+| Web   Main App    | Web + Native +   Dotnet       |
+
+- CLI
+  - Added support for importing Hybrid and Multi projects
+- VSCode
+  - Added support for importing Hybrid and Multi projects. Developer can import a Tizen.Native/Tizen.WEB/TizenDotnet Project to VSCode and then can build and pakcage.
+  - Added support for importing wgt(Tizen Web application package file) into Visual Studio Code Extension for Tizen
+- Visual Studio(Windows)
+  - Added support for importing Hybrid and Multi projects. Developer can import a Tizen.Native/Tizen.WEB/TizenDotnet Project to VS and then can build and pakcage.
+  - Added support for importing wgt(Tizen Web application package file) into Visual Studio Extension for Tizen
+
+
+### Fixed Bugs
+  - Fixed the issue in API Checker due to which warnings were not displayed for web apps in VS extension for Tizen.
+  - Fixed the error that occured on selecting TV profile while creating a new project in VS extension for Tizen.
+  - Fixed the failure to open yaml config files for native apps in VS extension for Tizen. 
+  - Fixed the issue due to which there was no option to import Tizen TV web app projects in VS & VSCode extension for Tizen.
+  - Fixed the issue in Import functionality and added missing support to import wgt(Web Application Package file) in VS & VSCode extension for Tizen
+
+### Deprecated
+  - Bridge Networking support in emulators will be deprecated in future Tizen Studio releases.
+
+
+### Known Issues
+
+- Installer
+  - In macOS, Tizen Studio related file are marked as Malware, hence it is recommended to select **Anywhere** under allow apps as described in https://www.imore.com/how-open-apps-anywhere-macos-catalina-and-mojave
+- Tizen Studio
+  - Valgrind profiling is not supported in Tizen 6.5 platform
+  - In macOS, if the UI perspectives are not displayed properly after updating Tizen Studio, it is recommended to restart the IDE. This issue is because of the uncleared cache from the previous installation.
+  - In case if you see any old perspective, it is recommended to create a new workspace and import the projects to the new workspace to resolve the issue.
+  - In macOS from catalina and above versions, Native Templates-5.5 will not build with CLI when compiler is set to **gcc**.
```

Comment:
Since “Catalina” is a proper noun specifying a macOS version, it should be capitalized. 
I am not sure why the words “Native Templates” have been capitalized here. Can you please confirm this? 
An article “the” is missing before the word “compiler”. 

```suggestion
  - In macOS from Catalina and above versions, Native Templates-5.5 will not build with CLI when the compiler is set to **gcc**.
```